documentation/components/tutorial-flow-components-setup.asciidoc

```diff
@@ -139,9 +109,29 @@ When using Maven you can add a the button component into your `pom.xml` as this:
 </dependencies>
 ----
 
-== Maven configuration to exclude webjars
+== Using WebJars
+
+Individual component packages consists of two main sections:
+
+. Java API classes.
+. Web Component files (HTML, JavaScript, CSS ..).
+
+Both sections are provided in the same JAR by default, and both sections are used to display the components correctly in pages.
```

Comment:
This part needs some rewriting. java api and the web component are distrubuted in different jar files. And both are needed. vaadin-foobar-flow depends on vaadin-foobar, so developers will automatically get the vaadin-foobar to their project, just by defining they want vaadin-foobar-flow to their application. 

One option would be to remove this line altogether.